pkgs/development/python-modules/arviz/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pandas
+, netcdf4
+, xarray
+, matplotlib
+, setuptools
+, numpy
+, scipy
+, pymc3
+#, pystan
+, emcee
+}:
+
+buildPythonPackage rec {
+  pname = "arviz";
+  version = "0.5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "04myd4m9rd5vkwxns5s15r0krfs6pdc04vfkm8kbs7d6rp69d9xh";
+  };
+  
+  propagatedBuildInputs = [
+    # needed to install
+    pandas
+    netcdf4
+    xarray
+    matplotlib
+    # needed to import
+    setuptools
+    # not needed to import, but used by many functions
+    # and is listed as a dependency in the documentation
+    numpy
+    scipy
+  ];
+
+  checkInputs = [
+    pymc3
+    #pystan
+    emcee
+  ];
+
+  # check requires pymc3, which is currently broken, and pystan, which is not
+  # yet packaged
+  doCheck = false;
```

do you mind selecting a subset of the tests?
```suggestion
  checkPhase = ''
    pytest arviz/tests/test_{utils,data}.py
  '';
```

---

Good idea, but data fails because it's trying to create directories in /homeless_shelter

---

HOME by default get set to "/homeless_shelter"

---

```suggestion
    HOME=$TMPDIR pytest arviz/tests/test_{utils,data}.py
```

---

That does fix the issue with the home directory, but there's also a problem that the saved_models directory was stripped out of the pypi distribution (and they don't do github releases or github version tagging after 0.3.2), and the last four tests rely on those models.

Any ideas?

---

In these cases I open an issue explaining why it's useful to remember to tag releases and ask them to start again.

---

Turns out they are tagging releases, and I'm just an idiot.